src/components/ContentNode/Reference.vue

```diff
@@ -91,6 +91,10 @@ export default {
       type: String,
       required: false,
     },
+    ariaRole: {
+      type: String,
+      default: null,
```

Comment:
Minor, non-blocking thing: the `null` default probably isn't needed if this is already an optional prop.